app/Libraries/UserVerificationState.php

```diff
@@ -119,7 +108,7 @@ public function markVerified()
         $this->session->put('verified', true);
         $this->session->save();
 
-        UserSessionEvent::newVerified($this->user->getKey(), $this->session->getKey())->broadcast();
+        UserSessionEvent::newVerified($this->user->getKey(), $this->session->getId())->broadcast();
```

Comment:
this needs to the cache-prefixed key because that's what osu-notification-server still uses and checks against; the same goes for logout; the `verified` event currently isn't being relayed to the client.